molecule/templates/vagrantfile.j2

```diff
@@ -40,16 +40,20 @@ Vagrant.configure('2') do |config|
     {%- if provider.type is defined and provider.type == 'virtualbox' and provider.name == current_provider %}
   config.vm.provider 'virtualbox' do |vb|
     {%- if provider.options %}
-      {%- for k, v in provider.options.iteritems()|sort %}
+      {%- for k, v in provider.options.iteritems() | sort %}
+        {%- if v == true or v == false %}
```

Comment:
Actually, I am doing a boolean comparison here.
for an input like
```
  providers:
    - name: virtualbox
      type: virtualbox
      options:
        linked_clone: false
```
in molecule.yml, the value of `v` will be `False`, which is a jinja boolean value. So I am comparing it with jinja `true` and `false`